x-pack/plugins/maps/public/connected_components/right_side_controls/layer_control/layer_toc/toc_entry/toc_entry_actions_popover/toc_entry_actions_popover.tsx

```diff
@@ -6,22 +6,27 @@
  */
 
 import React, { Component } from 'react';
-
 import { EuiPopover, EuiContextMenu, EuiIcon } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
 import { ILayer } from '../../../../../../classes/layers/layer';
 import { TOCEntryButton } from '../toc_entry_button';
 import {
   getVisibilityToggleIcon,
   getVisibilityToggleLabel,
-  EDIT_LAYER_LABEL,
+  EDIT_LAYER_SETTINGS_LABEL,
   FIT_TO_DATA_LABEL,
+  EDIT_FEATURES_LABEL,
 } from '../action_labels';
+import { ESSearchSource } from '../../../../../../classes/sources/es_search_source';
+import { VectorLayer } from '../../../../../../classes/layers/vector_layer';
+import { SCALING_TYPES } from '../../../../../../../common';
 
 export interface Props {
   cloneLayer: (layerId: string) => void;
+  enableShapeEditing: (layerId: string) => void;
+  enablePointEditing: (layerId: string) => void;
   displayName: string;
-  editLayer: () => void;
+  layerSettings: () => void;
```

Comment:
How about renaming to "openLayerSettings" to better capture what the callback does?